docker-compose.yml

```diff
@@ -334,23 +247,35 @@ services:
-      /entrypoint.sh call --json_input --json_output localhost:50151 CreateBridgePort    "{\"bridge_port\" : {\"spec\" : {\"mac_address\" : \"qrvMAAAB\", \"ptype\": \"TRUNK\" , \"logical_bridges\": [] }}, \"bridge_port_id\" : \"eth3\" }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateVrf            "{\"vrf_id\"            : \"blue\",     \"vrf\" :             {\"spec\" : {               \"loopback_ip_prefix\": {\"addr\": {\"af\": \"IP_AF_INET\", \"v4_addr\": 167772162}, \"len\": 24}, \"vtep_ip_prefix\": {\"addr\": {\"af\": \"IP_AF_INET\", \"v4_addr\": 167772162}, \"len\": 24} }} }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateVrf            "{\"vrf_id\"            : \"green\",    \"vrf\" :             {\"spec\" : {\"vni\" : 100, \"loopback_ip_prefix\": {\"addr\": {\"af\": \"IP_AF_INET\", \"v4_addr\": 167772162}, \"len\": 24}, \"vtep_ip_prefix\": {\"addr\": {\"af\": \"IP_AF_INET\", \"v4_addr\": 167772162}, \"len\": 24} }} }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateVrf            "{\"vrf_id\"            : \"yellow\",   \"vrf\" :             {\"spec\" : {\"vni\" : 101, \"loopback_ip_prefix\": {\"addr\": {\"af\": \"IP_AF_INET\", \"v4_addr\": 167772162}, \"len\": 24}, \"vtep_ip_prefix\": {\"addr\": {\"af\": \"IP_AF_INET\", \"v4_addr\": 167772162}, \"len\": 24} }} }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateLogicalBridge  "{\"logical_bridge_id\" : \"vni10\",    \"logical_bridge\" :  {\"spec\" : {\"vni\" : 10, \"vlan_id\": 10 }} }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateLogicalBridge  "{\"logical_bridge_id\" : \"vni20\",    \"logical_bridge\" :  {\"spec\" : {\"vni\" : 20, \"vlan_id\": 20 }} }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateLogicalBridge  "{\"logical_bridge_id\" : \"vni30\",    \"logical_bridge\" :  {\"spec\" : {\"vni\" : 30, \"vlan_id\": 30 }} }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateLogicalBridge  "{\"logical_bridge_id\" : \"vni40\",    \"logical_bridge\" :  {\"spec\" : {\"vni\" : 40, \"vlan_id\": 40 }} }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateLogicalBridge  "{\"logical_bridge_id\" : \"vni50\",    \"logical_bridge\" :  {\"spec\" : {              \"vlan_id\": 50 }} }" && \
+      /entrypoint.sh call --json_input --json_output localhost:50151 CreateSvi            "{\"svi_id\"            : \"vlan20\",   \"svi\" :             {\"spec\" : {\"vrf\": \"//network.opiproject.org/vrfs/blue\",   \"logical_bridge\": \"//network.opiproject.org/bridges/vni20\", \"mac_address\" : \"qrvMAAAh\", \"gw_ip_prefix\": [{\"addr\": {\"af\": \"IP_AF_INET\", \"v4_addr\": 336860161}, \"len\": 24}] }} }" && \
```

Comment:
I would prefer to name the SVI objects "<VRF>-<LB>" so that the association between VRF and logical bridge that they create is clear already from the name. The id of the SVI object on the gRPC API does not necessarily have to be the same as the interface name created in Linux. The latter is an implementation detail.

Reply:
Sorry, my comment got screwed up by the md interpreter. An SVI should be called e.g. `green-vlan10` if it connects VRF  `green` to logical bridge `vlan10`.

Reply:
thanks. You can use `back quotes` @JanScheurich to avoid formatting :-)

Reply:
sure